city_scrapers/spiders/wayne_land_bank.py

```diff
@@ -0,0 +1,76 @@
+import re
+from datetime import datetime
+
+from city_scrapers_core.constants import BOARD
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+from scrapy.http import FormRequest
+
+
+class WayneLandBankSpider(CityScrapersSpider):
+    name = "wayne_land_bank"
+    agency = "Wayne County Land Bank"
+    timezone = "America/Detroit"
+    allowed_domains = ["public-wclb.epropertyplus.com"]
+    start_urls = ["https://public-wclb.epropertyplus.com/landmgmtpub/app/base/customPage"]
+    location = {
+        "name": "Wayne County Treasurer's Office",
+        "address": "400 Monroe St 5th Floor, Detroit, MI 48226, United States",
+    }
+
+    def start_requests(self):
+        yield FormRequest(
+            self.start_urls[0],
+            formdata={
+                "templateName": "BoardofDirectors",
+                "title": "Board+of+Directors"
+            },
+            callback=self.parse,
+            dont_filter=True
+        )
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        data = response.xpath('//script[@id="custom_page_template"]/text()').extract()
+        data = re.findall(
+            '(Jan(?:uary)|Feb(?:ruary)|Mar(?:ch)|Apr(?:il)|May|Jun(?:e)|Jul(?:y)'
+            '|Aug(?:ust)|Sep(?:tember)|Oct(?:ober)|Nov(?:ember)|Dec(?:ember))'
+            r'(.*?)(?=\.\s|\\n)'
+            '(.*?)(?=\\n)', data[0]
+        )
+
+        for item in data:
+            meeting = Meeting(
+                title=item[2].strip(" .()"),
+                description="",
+                classification=BOARD,
+                start=self._parse_start("".join(item[0:2])),
+                end=None,
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(""),
```

Comment:
The `_parse_location` method here isn't necessary, but a `_validate_location` method would be good here to check if the location has changed. https://github.com/City-Bureau/city-scrapers/blob/c23f1fd17b1ced1c01dff295c8cdf1ca191f0244/city_scrapers/spiders/chi_ssa_34.py#L58-L60